content/docs/apps/tic.md

```diff
@@ -0,0 +1,14 @@
+---
+menu:
+  docs:
+    parent: advanced
+title: Restricting access
+---
+
+To comply with the [TIC Initiative](https://www.dhs.gov/trusted-internet-connections), you can configure cloud.gov to restrict access to cloud.gov system services to a set of IP address ranges. This restriction prevents users from your email domain (e.g., `gsa.gov`) from accessing cloud.gov services unless their requests originate from one of your whitelisted IP address ranges.
+
+To configure IP address restriction, create a [support ticket](mailto:cloud-gov-support@gsa.gov?body=Email%20domain%3A%0A%0AEgress%20IP%20ranges%3A%0A%0AAgency%20CIO%3A%0A). Because address restriction applies to all cloud.gov users within your email domain, we will confirm your request with your agency CIO before changing configuration.
```

Comment:
Trying to think of ways to make this easier - can we give them an email template that includes asking them to cc their agency CIO on the email request, which instructs the CIO to reply-all with approval?

Reply:
This is a (very minimal) email template that should render as:

```
Email domain:

Egress IP ranges:

Agency CIO:
```

We could add a note to the template text asking the user to cc their CIO. Help drafting that text would be appreciated 😃